### PR TITLE
Feature/ship repairs minigame 

### PR DESCRIPTION
**Pull request by maxwellpark**

### CODE REVIEW

File: Assets/Scripts/Minigames/ShipRepairs/RepairsManager.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        feedbackText = string.Empty;
+    }
+
+    private void OnGUI()
+    {
+        var localStyle = new GUIStyle();
+        localStyle.normal.textColor = Color.black;
+
+        GUI.Label(new Rect(
```
Comment (ambid17) [inline Assets/Scripts/Minigames/ShipRepairs/RepairsManager.cs]: AHHHH NO

File: Assets/Scripts/Minigames/ShipRepairs/Workstation.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    public bool isRotating;
+
+    private float rotationSpeed;
+    private float startingSpeed; 
+    private float speedIncrease;
+
+
+    private void Start()
+    {
+        startingSpeed = 3f;
```
Comment (ambid17) [inline Assets/Scripts/Minigames/ShipRepairs/Workstation.cs]: Static constant?

File: Assets/Scripts/Minigames/ShipRepairs/Workstation.cs [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        }
+
+        if (isRotating)
+        {
+            RotateWorkStation();
+        }
+    }
+
+    private void OnGUI()
+    {
```
Comment (ambid17) [inline Assets/Scripts/Minigames/ShipRepairs/Workstation.cs]: Extraneous

File: Assets/Scripts/Minigames/ShipRepairs/Workstation.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+    private void ResetRotationSpeed()
+    {
+        rotationSpeed = startingSpeed;
+    }
+
+    // This will increase the difficulty by disorientating the player 
+    public void ReverseRotationDirection()
+    {
+        rotationSpeed = rotationSpeed > 0 ? rotationSpeed * -1 : Mathf.Abs(rotationSpeed);
```
Comment (ambid17) [inline Assets/Scripts/Minigames/ShipRepairs/Workstation.cs]: Just multiply rotationSpeed by -1 each time and it’ll flip

**Comment by maxwellpark** [comment]
I might amalgamate RepairsManager and Workstation. The concerns are split between them in an awkward way. Then have a separate class for the data and/or presentation.

**Comment by maxwellpark** [comment]
The functionality itself should be ok though.

**Review by ambid17 — APPROVED** [review]

**Pull request merged**